Automattic/wp-calypso
# Support: Add state for Support Level

#### Changes proposed in this Pull Request

From the Chat configuration endpoint, receive a new property `supportLevel` and save it in Redux state.

This PR does not make use of this property, but prepares it to be ready to ship on quick notice, for example to show Contact Form notices only to users with certain levels of support.

#### Testing instructions
Go to http://calypso.localhost:3000/help/contact, and see the "Limited chat availability" notice at the top.

Change [this line](https://github.com/Automattic/wp-calypso/pull/45207/files#diff-bfbe01f0a66b1aa6a6708379328f1d66R55) to:
```
import getSupportLevel, { SUPPORT_LEVEL_PERSONAL } from 'state/selectors/get-support-level';
```

Change [this line](https://github.com/Automattic/wp-calypso/blob/master/client/me/help/help-contact/index.jsx#L600) to:
```
{ isUserAffectedByLiveChatClosure && this.props.supportLevel === SUPPORT_LEVEL_PERSONAL && (
```

Go back to http://calypso.localhost:3000/help/contact — you should only see the notice if the highest paid plan you own or are an admin of is Personal. (Try this with the other support levels as well.)
